libs/wire-api/src/Wire/API/Routes/Public/Brig.hs

```diff
@@ -135,6 +135,15 @@ data Api routes = Api
         :> "self"
         :> ReqBody '[JSON] DeleteUser
         :> MultiVerb 'DELETE '[JSON] DeleteSelfResponses (Maybe Timeout),
+    updateUserEmailValidation ::
+      routes
+        :- Summary "Resend email validation email."
+        :> Description "If the user has a pending email validation, the validation email will be resent."
+        :> ZUser
+        :> "users"
+        :> CaptureUserId "uid"
```

Comment:
> In my limited understanding of the subject, this will force you to handle the qualifying domain in which the user lives.  In this end-point, we can throw an error "only supported for local domain" here.

That's correct, and of course it also makes the API require a domain. In galley there is an `ensureLocal` helper which throws `federationNotImplemented` if the resulting qualified value is not local, and otherwise converts it into a `Local x` value (i.e. a qualified value with a guarantee that its domain is local).

Reply:
Misunderstanding: we don't want to implement the federated version later, we want to make sure that this can only be called from local.  Does that mean keep the code as it is?

(Your plan would be better though if we plan to have federated teams, where some members are on other instances than others.  But I don't think that's a plan, right?)

Reply:
Ah, sorry, I haven't actually looked at what this endpoint is about. :) If it doesn't make sense for remote values, then I guess you shouldn't use the qualified version. You're right, federated teams are not a thing for the time being.

Reply:
@fisx please resolve this conversation if I can leave it like this.